src/Installer/Microsoft.Dotnet.Installation/Internal/ReleaseManifest.cs

```diff
@@ -308,348 +133,145 @@ private List<Product> GetProductsForMajorVersion(ProductCollection index, int ma
     /// <param name="isLts">True for LTS (Long-Term Support), false for STS (Standard-Term Support)</param>
     /// <param name="mode">InstallComponent.SDK or InstallComponent.Runtime</param>
     /// <returns>Latest stable version string matching the support status, or null if none found</returns>
-    private ReleaseVersion? GetLatestVersionBySupportStatus(ProductCollection index, bool isLts, InstallComponent component)
+    private static ReleaseVersion? GetLatestVersionBySupportStatus(IEnumerable<Product> index, ReleaseType releaseType, InstallComponent component)
```

Comment:
Would it be more understandable to use the same terminology in teh method name as in the parameter that's passed in (ReleaseType)?

```suggestion
    private static ReleaseVersion? GetLatestVersionByReleaseType(IEnumerable<Product> index, ReleaseType releaseType, InstallComponent component)
```

Reply:
Seems wise to me :) Release Type is a term only used in the deployment library so I didn't initially choose it and chose to respect the manifest term, but this makes sense.